front/app/containers/ProjectsShowPage/timeline/VotingResults/VotingResultCard.tsx

```diff
@@ -234,7 +234,7 @@ const VotingResultCard = ({ idea, phaseId, rank }: Props) => {
           </Box>
         )}
 
-        <Header className="e2e-card-title">
+        <Header>
```

Comment:
Just removed this class to avoid confusion and because it wasn't used for anything